db/primary_migrate/20241017153042_rename_socure_docv_token_in_document_capture_session.rb

```diff
@@ -0,0 +1,5 @@
+class RenameSocureDocvTokenInDocumentCaptureSession < ActiveRecord::Migration[7.1]
+  def change
+    safety_assured { rename_column :document_capture_sessions, :socure_docv_token, :socure_docv_transaction_token }
```

Comment:
I wish [StrongMigrations' documentation](https://github.com/ankane/strong_migrations?tab=readme-ov-file#renaming-a-column) included more details than just "will cause errors", but based on their "safer approach" guidelines, I do think we'd be safe if we're not currently using this column.

Reply:
There's also a [`rename_column_safely`](https://github.com/ankane/strong_migrations/pull/106/files#diff-b435a6690c2a730abf1a2ea884cc6299cb601c86d678c06741a6b5623add6ae4R153) helper from StrongMigrations, which preserves the expressivity while doing it the way they recommend. Based on previous comment, don't know that it's strictly necessary.